SCRAPER_INFO.md

```diff
@@ -11,7 +11,7 @@
 - **Current Status:** Currently is pulled correctly by scraper.
 
 ### Date
-- **Function name: ** `get_date()`
+- **Function name: ** `decision_date()`
```

Comment:
The function still name get_date() because u haven't change it in ocr.py

Reply:
Oh my bad, I might have accidently wrote that over when using ctrl-f in the md file. I should just revert that right?

Reply:
Yes Waqas